xrootd/client/file.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package client // import "go-hep.org/x/hep/xrootd/client"
+
+import "go-hep.org/x/hep/xrootd/xrdfs"
+
+// File implements access to a content and meta information of file over XRootD.
+type file struct {
+	c           *Client
```

Comment:
instead of `*Client`, wouldn't it make more sense to have `*fileSystem` instead?

Reply:
From some point of view, it is so, but I can't come up with the scenario when `fileSystem` will be needed in the `file` yet.

Reply:
ok. well, one can always reach at the underlying `Client` from `*fileSystem` and get a `*fileSystem` from `Client` (although, in that case, at the price of a type-check.)

let's go with `fs *fileSystem` instead?